recipe/meta.yaml

```diff
@@ -134,7 +140,8 @@ about:
   home: http://sources.redhat.com/glibc/
   license: LGPL-2.0-or-later AND LGPL-2.0-or-later WITH exceptions AND GPL-2.0-or-later
```

Comment:
Can you fix this too?

Reply:
For sure but idk which exception to put in there.

Reply:
```suggestion
  license: LGPL-2.0-or-later AND LGPL-2.0-or-later WITH exceptions AND GPL-2.0-or-later AND MPL-2.0
```

Reply:
Ohhh I thought you meant the "WITH exceptions" which is not proper SPDX.